lib/parse.js

```diff
@@ -194,7 +194,7 @@ JS_Parse_Error.prototype.toString = function() {
 };
 
 function js_error(message, filename, line, col, pos) {
-    throw new JS_Parse_Error(message, line, col, pos);
+    throw new JS_Parse_Error(''+ message + ' at ' + filename, line, col, pos);
```

Comment:
Replace the single quotes with double quotes because all other code uses double quotes.